core/src/main/scala/org/apache/spark/h2o/H2OConf.scala

```diff
@@ -27,7 +28,7 @@ import org.apache.spark.repl.h2o.H2OInterpreter
   * Configuration holder which is representing
   * properties passed from user to Sparkling Water.
   */
-class H2OConf(@transient val sc: SparkContext) extends Logging with InternalBackendConf {
+class H2OConf(@transient val sc: SparkContext) extends Logging with ExternalBackendConf with InternalBackendConf {
```

Comment:
At least you do not need to store SparkContext as field, just extracting SparkConf should be enough.